src/Security/Authentication/Cookies/src/ITicketStore.cs

```diff
@@ -18,6 +19,14 @@ public interface ITicketStore
         /// <returns>The key that can be used to retrieve the identity later.</returns>
         Task<string> StoreAsync(AuthenticationTicket ticket);
 
+        /// <summary>
+        /// Store the identity ticket and return the associated key.
+        /// </summary>
+        /// <param name="ticket">The identity information to store.</param>
+        /// <param name="cancellationToken"></param>
```

Comment:
```suggestion
        /// <param name="cancellationToken">The <see cref="CancellationToken"/> used to propagate notifications that the operation should be canceled.</param>
```
You can use this comment for `cancellationToken` parameter.